neuralpredictors/layers/readouts/factorized.py

```diff
@@ -19,12 +23,31 @@ def __init__(
         init_noise=1e-3,
         constrain_pos=False,
         positive_weights=False,
+        positive_spatial=True,
```

Comment:
```suggestion
        positive_spatial=False,
```
Reasoning: I think we should not change the default behavior of code if we do not have too, as this would break models we already have trained / result in reproducibility issues.